mpz-circuits/src/tracer.rs

```diff
@@ -50,3 +50,10 @@ where
         BinaryRepr::Array(tracer.into_iter().map(|tracer| tracer.into()).collect())
     }
 }
+
+impl<'a> Tracer<'a, Bit> {
+    /// Returns the single node associated with the bit.
+    pub fn node(&self) -> Node<Feed> {
+        self.to_inner().nodes()[0]
+    }
+}
```

Comment:
So that `.to_inner().nodes()[0]` could be replaced with `.node()`, otherwise repeated many times now.